src/hub.html

```diff
@@ -153,10 +158,12 @@
                         <template data-name="Spine">
                             <a-entity personal-space-invader="radius: 0.2; useMaterial: true;" bone-visibility>
                                 <a-entity billboard matrix-auto-update visibility-while-frozen="withinDistance: 10; requireHoverOnNonMobile: false;">
-                                    <a-entity hoverable mixin="rounded-text-button" block-button ui-class-while-frozen position="0 0 .35"> </a-entity>
-                                    <a-entity text="value:Hide; width:2.5; align:center;" position="0 0 0.36"></a-entity>
-                                    <a-entity hoverable mixin="rounded-text-button" visible-if-permitted="kick_users" kick-button ui-class-while-frozen position="0 -0.25 .35"> </a-entity>
-                                    <a-entity text="value:Kick; width:2.5; align:center;"  visible-if-permitted="kick_users" position="0 -0.25 0.36"></a-entity>
+                                    <a-entity hoverable mixin="rounded-text-button" block-button ui-class-while-frozen position="0 0 .35">
```

Comment:
Better merge fast unless you want to be the one to resolve 100 merge conflicts
https://github.com/mozilla/hubs/compare/try-replacing-super-hands?expand=1#diff-a796e11be3d0dfc0f893148f1086736dR156